MIG-1552: Fix must-gather image in MTC 1.7 docs (#75087)
openshift/openshift-docs: modules/migration-using-must-gather.adoc

. Navigate to the directory where you want to store the `must-gather` data.
. Run the `oc adm must-gather` command for one of the following data collection options:

> ifdef::troubleshooting-3-4[]

Author: ```suggestion
ifdef::troubleshooting-3-4,troubleshooting-mtc[]
```

. Navigate to the directory where you want to store the `must-gather` data.
. Run the `oc adm must-gather` command for one of the following data collection options:

ifdef::troubleshooting-3-4[]

> * To collect data for the past hour, use the following command:

Reviewer: ```suggestion
* To collect data for the past hour, enter the following command:
```

+
[source,terminal]
----
$ oc adm must-gather --image=registry.redhat.io/rhmtc/openshift-migration-must-gather-rhel8:v1.7
----
+
This command saves the data as the `must-gather/must-gather.tar.gz` file. You can upload this file to a support case on the link:https://access.redhat.com/[Red Hat Customer Portal].

> * To collect data for the past 24 hours, use the following command:

Reviewer: ```suggestion
* To collect data for the past 24 hours, enter the following command:
```

. Navigate to the directory where you want to store the `must-gather` data.
. Run the `oc adm must-gather` command for one of the following data collection options:

ifdef::troubleshooting-3-4[]

> * To collect data for the past hour:

Reviewer 2: This really should be a more complete sentence, such as "To collect data for the past hour, use the following command:" (replicating what's in the text later, lines 61ff)

Reviewer: Not gonna hold up a merge review for this bc it's in our docs as-is, but it should really be "run" or "enter" the following command. Based on https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command

> * To collect data for the past hour:

Author: ```suggestion
* To collect data for the past hour, use the following command:
```

+
[source,terminal]
----
$ oc adm must-gather --image=registry.redhat.io/rhmtc/openshift-migration-must-gather-rhel8:v1.7
----
+

> The data is saved as `must-gather/must-gather.tar.gz`. You can upload this file to a support case on the link:https://access.redhat.com/[Red Hat Customer Portal].

Author: ```suggestion
This command saves the data as the `must-gather/must-gather.tar.gz` file. You can upload this file to a support case on the link:https://access.redhat.com/[Red Hat Customer Portal].
```

> * To collect data for the past 24 hours:

Author: ```suggestion
* To collect data for the past 24 hours, use the following command:
```

+
[source,terminal]
----
$ oc adm must-gather --image=registry.redhat.io/rhmtc/openshift-migration-must-gather-rhel8:v1.7 -- /usr/bin/gather_metrics_dump
----
+

> This operation can take a long time. The data is saved as `must-gather/metrics/prom_data.tar.gz`.

Reviewer 2: I would remove the passive voice and make the second sentence more complete:

This command saves the data as the `must-gather/metrics/prom_data.tar.gz` file.

Reviewer 2: If you do decide to take up my style suggestions, please also apply them to the currently-unchanged lines right after these ones, in order to ensure consistency between builds of this source.

> This operation can take a long time. The data is saved as `must-gather/metrics/prom_data.tar.gz`.

Author: ```suggestion
This operation can take a long time. This command saves the data as the `must-gather/metrics/prom_data.tar.gz` file.
```

endif::[]
ifdef::troubleshooting-mtc[]

> * To collect data for the past hour:

Author: ```suggestion
* To collect data for the past 24 hours, use the following command:
```
